showroom/models.py

```diff
@@ -0,0 +1,120 @@
+from django.db import models
+from django_countries.fields import CountryField
+from django.contrib.auth.models import User
+
+
+class CarManufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Car(models.Model):
+    CAR_TYPE_CHOICES = [
+        ('Sedan', 'Sedan'),
+        ('Coupe', 'Coupe'),
+        ('Crossover', 'Crossover'),
+        ('Hatchback','Hatchback'),
+        ('Minivan', 'Minivan'),
+    ]
+    manufacturer = models.ForeignKey(CarManufacturer, on_delete=models.CASCADE)
+    car_type = models.CharField(max_length=100, choices=CAR_TYPE_CHOICES,verbose_name="Вид авто")
+    model = models.CharField(max_length=150, verbose_name="Модель авто")
+    year = models.IntegerField(default=0,verbose_name="Год выпуска")
+    price = models.FloatField(verbose_name="Цена авто")
+    is_active = models.BooleanField(default=True)
+    time_create = models.DateTimeField(auto_now_add=True)
+    time_update = models.DateTimeField(auto_now=True)
+
```

Comment:
it is better to place created_at and updated_at fields to a separate abstract model. And inherit that model in all necessary model